input/fsh/SD_RadiotherapyPlanPrescription.fsh

```diff
@@ -6,12 +6,15 @@ Title: "Radiotherapy Plan Prescription"
 Description: "A Radiotherapy Plan Prescription is a request for radiotherapy treatment with a single treatment plan."
 * insert RadiotherapyPhaseAndPlanPrescriptionCommon
 * code = SnomedRequestedCS#USCRS-34602 "Radiotherapy treatment plan (regime/therapy)"
-* insert BasedOnSlicing
+* insert OpenProfileBasedSlicing(basedOn)
 * basedOn contains
     course-prescription 0..1 MS and
     phase-prescription 0..1 MS
 * basedOn[course-prescription] ^short = "Radiotherapy Course Prescription that this Radiotherapy Plan Prescription fulfills (often together with other treatment phases)."
 * basedOn[phase-prescription] ^short = "Radiotherapy Phase Prescription that this Radiotherapy Plan Prescription fulfills (often together with other treatment phases)."
 * basedOn[course-prescription] only Reference(RadiotherapyCoursePrescription)
 * basedOn[phase-prescription] only Reference(RadiotherapyPhasePrescription)
-* replaces only Reference(RadiotherapyPlanPrescription)
\ No newline at end of file
+* replaces only Reference(RadiotherapyPlanPrescription)
+* performer only Reference(RadiotherapyTreatmentDevice or RadiotherapySeedDevice)
+* performer MS
+* performer ^short = "RadiotherapyTreatment or Seed Devices used as part of therapy."
```

Comment:
performer restrictions should be removed here. The come with the ruleset, current PrescriptionCommon, but should be rather RequestCommon as suggested above. "only reference on performer is too restrictive. Only reference on replaces I would leave here, because there is really nothing else that can be replaced by a Plan Prescription. Same for the other requests.